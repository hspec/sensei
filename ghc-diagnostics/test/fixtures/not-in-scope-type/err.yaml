version: '1.1'
ghcVersion: ghc-9.10.0
span:
  file: test/fixtures/not-in-scope-type/Foo.hs
  start:
    line: 2
    column: 8
  end:
    line: 2
    column: 11
severity: Error
code: 76037
message:
- 'Not in scope: type constructor or class `Map'''
hints: []
