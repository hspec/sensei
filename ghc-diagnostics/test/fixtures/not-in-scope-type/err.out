test/fixtures/not-in-scope-type/Foo.hs:2:8: error: [GHC-76037]
    Not in scope: type constructor or class `Map'

